assignment/views/homework_view.xml

```diff
@@ -213,17 +193,13 @@
         </record>
 
         <!-- Menu items for Assignment-->
-
         <menuitem id="menu_assignment" name="Assignment" parent="school.menu_ems" sequence="5"/>
-
         <menuitem id="menu_student_assignment_sub_form" name="Student Assignment"
                 parent="menu_assignment"
                 action="action_school_student_assignment_form" sequence="2"
                 groups="school.group_school_administration,school.group_school_teacher,school.group_school_student"/>
-
          <menuitem id="menu_teacher_assignment_sub_form" name="Teacher Assignment"
                 parent="menu_assignment"
                 action="action_school_teacher_assignment_form"
                 groups="school.group_school_administration,school.group_school_teacher"/>
-
 </odoo>
```

Comment:
Dear @arpit983 , what is the reason of removing blank lines here, they are an important part of code as a good programming practice.